src/components/Swap/Toolbar/index.tsx

```diff
@@ -61,7 +61,7 @@ function CaptionRow() {
       default:
     }
 
-    if (state === TradeState.LOADING) {
+    if (state === TradeState.LOADING && trade == null) {
```

Comment:
You should always use `===` to avoid bugs from implicit casting. (I thought this was a lint rule 🤷 )

Reply:
i actually wanted `==` because i'm checking for `undefined`

Reply:
but i can make it explicitly check for `undefined`